modules/materials/materials.model.js

```diff
@@ -0,0 +1,18 @@
+const materialType = ` 
```

Comment:
Please, rename file to `material.graphql.js`

Reply:
done